gravitee-apim-console-webui/src/management/application/details/subscriptions/subscription/application-subscription.component.html

```diff
@@ -0,0 +1,109 @@
+    </mat-card-header>
+    <mat-card-content>
+      <dl class="gio-description-list">
+        <dt>ID</dt>
+        <dd gioClipboardCopyWrapper [contentToCopy]="pageVM.subscription.id">
+          {{ pageVM.subscription.id || '-' }}
+        </dd>
+
+        <dt>API</dt>
+        <dd gioClipboardCopyWrapper [contentToCopy]="pageVM.subscription.api.name + ' - ' + pageVM.subscription.api.version">
+          {{ pageVM.subscription.api.name }} - {{ pageVM.subscription.api.version }}
+        </dd>
+
+        <dt>Plan</dt>
+        <dd gioClipboardCopyWrapper [contentToCopy]="pageVM.subscription.plan.name">
+          {{ pageVM.subscription.plan.name || '-' }}
+        </dd>
+
+        <dt>Status</dt>
+        <dd gioClipboardCopyWrapper [contentToCopy]="pageVM.subscription.status | uppercase">
+          {{ (pageVM.subscription.status | uppercase) || '-' }}
+        </dd>
+
+        <dt>Subscribed by</dt>
+        <dd gioClipboardCopyWrapper [contentToCopy]="pageVM.subscription.subscribed_by.displayName">
+          {{ pageVM.subscription.subscribed_by.displayName || '-' }}
+        </dd>
+
+        @if (pageVM.subscription.request) {
+          <dt>Publisher message to subscriber</dt>
+          <dd gioClipboardCopyWrapper [contentToCopy]="pageVM.subscription.request">
+            {{ pageVM.subscription.request || '-' }}
+          </dd>
+        }
+
+        @if (pageVM.subscription.reason) {
+          <dt>Subscriber message to publisher</dt>
+          <dd gioClipboardCopyWrapper [contentToCopy]="pageVM.subscription.reason">
+            {{ pageVM.subscription.reason || '-' }}
+          </dd>
+        }
+
+        <dt>Created at</dt>
+        <dd>{{ (pageVM.subscription.created_at | date: 'medium') || '-' }}</dd>
+        <dt>Processed at</dt>
+        <dd>{{ (pageVM.subscription.processed_at || '-' | date: 'medium') || '-' }}</dd>
+
+        @if ('REJECTED' !== pageVM.subscription.status) {
+          <dt>Starting at</dt>
+          <dd>{{ (pageVM.subscription.starting_at | date: 'medium') || '-' }}</dd>
+          <dt>Paused at</dt>
+          <dd>{{ (pageVM.subscription.paused_at | date: 'medium') || '-' }}</dd>
+          <dt>Ending at</dt>
+          <dd>{{ (pageVM.subscription.ending_at | date: 'medium') || '-' }}</dd>
+        }
+        <dt>Closed at</dt>
+        <dd>{{ (pageVM.subscription.closed_at | date: 'medium') || '-' }}</dd>
+      </dl>
+    </mat-card-content>
+
+    @if (pageVM.subscription.status === 'ACCEPTED' || pageVM.subscription.status === 'PENDING' || pageVM.subscription.status === 'PAUSED') {
+      <mat-card-actions *gioPermission="{ anyOf: ['application-subscription-d'] }">
+        <button mat-raised-button color="warn" (click)="closeSubscription(pageVM.application, pageVM.subscription)">
+          <mat-icon svgIcon="gio:x-circle"></mat-icon>
+          Close
+        </button>
+      </mat-card-actions>
+    }
+  </mat-card>
+
+  <subscription-api-keys
```

Comment:
Does it make sens to display this API Key table no matter what is the plan security type or mode ?

For example on my push plan and JWT plan I have this

<img width="1792" alt="Capture d’écran 2024-04-16 à 09 14 22" src="https://github.com/gravitee-io/gravitee-api-management/assets/25704259/9b1d8311-c26d-4d47-9045-0d5bf362664e">

Reply:
👍  yes, I think that was probably the case before, wasn't it? I must have missed the condition.